JSON_Schemas/Weights_and_Condition_Scores/iscWeightEventResource.json

```diff
@@ -0,0 +1,53 @@
+{
+    "required": [
+        "weight",
+        "units"
+    ],
+
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "icarEventCoreResource.json"
+        },
+        {
+            "properties": {
+                "weight": {
+                    "type": "number",
+                    "format": "double",
+                    "description": "The weight observation, in the units specified (usually kilograms)"
+                },
+                "units": {
+                    "type": "string",
+                    "enum": ["KGM", "GRM", "LBR", "TNE", "MC", "MGM"],
+                    "default": "KGM",
+                    "description": "Units specified in UN/CEFACT 3-letter form. Default if not specified is KGM. Mass units for weight from UN/CEFACT trade facilitation recommendation 20.\r\n Kilogram, Gram, Pound, Metric Tonne, Microgram, Miligram."
```

Comment:
Any reason why we are redefining these here rather than using the uncefactMassUnitsType as per the ICAR definition?